manifests/init.pp

```diff
@@ -351,6 +355,9 @@
   if $keys {
     create_resources('apt::key', $keys)
   }
+  if $keyrings {
+    create_resources('apt::keyring', $keyrings)
```

Comment:
please don't use `create_resources()`. That's a deprecated pattern. Instead, iterate on `$keyrings`:

```puppet
$keyrings.each |$key, $data| {
  apt::keyring { $key:
    * => $data,
   }
}
```